custom_dc/build_image.md

```diff
@@ -18,27 +18,35 @@ While you are just testing out data changes, you don't need to build the website
 
 Data Commons provides two prebuilt images in the Google Artifact Registry that you can download to run in a Docker container:
 
-- `gcr.io/datcom-ci/datacommons-services:stable`. This is a tested, stable version but may be several weeks old.
-- `gcr.io/datcom-ci/datacommons-services:latest`. This is the latest version built from head.
+- `gcr.io/datcom-ci/datacommons-data:stable` and `gcr.io/datcom-ci/datacommons-services:stable`. This is a tested, stable version but may be several weeks old.
```

Comment:
nit: Since it's now talking about 2 images, perhaps pluralize the descriptions here and below.

Reply:
OK